cas/service.c

```diff
@@ -2148,6 +2150,24 @@ static int cas_kv_load_done(struct cas_fom *fom, enum m0_cas_opcode  opc,
 	return M0_RC(M0_FSO_AGAIN);
 }
 
+static bool cas_is_layout_ctg(struct m0_cas_op *op)
+{
+	struct m0_fid dix_fid;
+
+	if(op)
+	{
+		if(m0_dix_fid_validate_cctg(&op->cg_id.ci_fid))
+		{
```

Comment:
[Style] Can you combine the if statements like 
if (op &&  m0_dix_fid_validate_cctg(&op->cg_id.ci_fid)